process/track/sovereignBlockTrackerFactory.go

```diff
@@ -0,0 +1,45 @@
+package track
+
+import (
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-go/process"
+)
+
+// BlockTrackerCreator is an interface for creating block trackers
+type BlockTrackerCreator interface {
```

Comment:
Could be this moved in process/interface.go or process/track/interface.go?

Reply:
moved to process/track/interface.go